doc/source/user_guide/data_types.txt

```diff
@@ -73,6 +74,30 @@ issued::
    array([  0, 128, 255], dtype=uint8)
 
 
+Additionally, some functions take a ``preserve_range`` argument where a range
+conversion is convenient but not necessary. For example, interpolation in
+``transform.warp`` requires an image of type float, which should have a range
+in [0, 1]. So, by default, input images will be rescaled to this range.
+However, with ``preserve_range=True``, the original range of the data will be
```

Comment:
It would be good to add a concrete example.  E.g., that practitioners working with, say, temperature measurements or rainfall levels or what have you, would not want their values clipped.